ui/src/TestUserInput.ts

```diff
@@ -76,6 +76,26 @@ export class TestUserInput implements IAzureUserInput {
         throw new Error(`Unexpected call to showWarningMessage. Message: ${message}`);
     }
 
+    public showInformationMessage<T extends MessageItem>(message: string, ...items: T[]): Thenable<T | undefined>;
+    public showInformationMessage<T extends MessageItem>(message: string, options: MessageOptions, ...items: T[]): Thenable<T | undefined>;
+    // tslint:disable-next-line:no-any
+    public showInformationMessage<T extends MessageItem>(message: string, ...args: any[]): Thenable<T | undefined> {
+        if (args.length > 0) {
```

Comment:
Right now I don't consume any input if no items are passed to information message.  I think this makes most sense, but I could see other arguments.